examples/splinterdb_iterators_example.c

```diff
@@ -0,0 +1,168 @@
+// Copyright 2022 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/*
+ * SplinterDB Iterators example program, using start-key for scan.
+ */
+#include <stdio.h>
+#include <string.h>
+
+#include "splinterdb/default_data_config.h"
+#include "splinterdb/splinterdb.h"
+
+#define DB_FILE_NAME    "splinterdb_iterators_example_db"
+#define DB_FILE_SIZE_MB 1024 // Size of SplinterDB device; Fixed when created
+#define CACHE_SIZE_MB   64   // Size of cache; can be changed across boots
+
+/* Application declares the limit of key-sizes it intends to use */
+#define APP_MAX_KEY_SIZE ((int)100)
```

Comment:
Remove `APP_`

Reply:
To make this quicker to resolve ... I changed all tokens named `APP_` to `USER_<something>`. E.g.

```
 17 /* Application declares the limit of key-sizes it intends to use */
 18 #define USER_MAX_KEY_SIZE ((int)100)
 19
 20 /* Avg value size and max value sizes we expect to deal with in this program */
 21 #define USER_AVG_VALUE_SIZE ((int)128)
 22 #define USER_MAX_VALUE_SIZE ((int)1024)
 ```
 
 Hope this closes out this remark.